test/tests/transfers/multiapp_high_order_variable_transfer/tests

```diff
@@ -0,0 +1,42 @@
+[Tests]
+   design = "MultiAppMeshFunctionTransfer.md"
+   issues = "#13446"
+
+  [L2_Lagrange]
+    type = 'Exodiff'
+    input = 'master_L2_Lagrange.i'
+    exodiff = 'master_L2_Lagrange_out.e master_L2_Lagrange_out_sub0.e'
+    requirement = "The system shall support transfer first order elemental variable"
+  []
+
+  [L2_Lagrange_different_mesh]
+    type = 'Exodiff'
+    input = 'master_L2_Lagrange_diff.i'
+    exodiff = 'master_L2_Lagrange_diff_out.e master_L2_Lagrange_diff_out_sub0.e'
+    requirement = "The system shall support transfer first order elemental variable when a different mesh is used in sub"
+  []
+
+  [L2_Lagrange_nearest_node]
+    type = 'Exodiff'
+    input = 'master_L2_Lagrange_nearest_node.i'
+    exodiff = 'master_L2_Lagrange_nearest_node_out.e master_L2_Lagrange_nearest_node_out_sub0.e'
+    # It is well-known that MultiAppNearestNodeTransfer could generate different results with different numbers of cores
+    max_parallel = 1
+    requirement = "The system shall support transfer first order elemental variable with MultiAppNearestNodeTransfer"
```

Comment:
Instead of naming the object in the requirement, talk about what you are trying to do. In this case picking the nearest node instead of interpolating.